src/chrome/content/rules/Idenyt.dk.xml

```diff
@@ -1,20 +1,10 @@
-<!--
-	Cloudflare SSL on ^ and www
-
-	Problematic domains:
-		^ ²
-		www ²
-		foto ¹
-		jul2k8 ¹
-		(www.)tv ¹
```

Comment:
Why did you remove all comments? Some are still true.

Reply:
@J0WI not sure actually, fixed.